lib/clif-backend/src/resolver.rs

```diff
@@ -52,6 +53,10 @@ impl FuncResolverBuilder {
             memory.protect(0..memory.size(), Protect::ReadWrite)?;
         }
 
+        for i in unsafe { memory.as_slice_mut() } {
+            *i = 0xCC;
```

Comment:
I'm curious to know what this is doing? Is a way to "clear" the memory?

Reply:
Normally, the empty space in the executable-mapped memory would be filled with zeros. These are actually valid x86 instructions, so this fills the empty space with `int 3`, which will trap.